components/disk/component.go

```diff
@@ -682,7 +665,16 @@ func (c *component) fetchNFSPartitions(cr *checkResult) bool {
 			continue
 		}
 
-		cr.NFSPartitions = parts
+		// possible that some partitions were just un-mounted
+		// thus stat call times out
+		for _, p := range parts {
+			if p.StatTimedOut {
+				log.Logger.Warnw("stat timed out for nfs partition", "device", p.Device, "mount_point", p.MountPoint)
+				continue
```

Comment:
Why do you hide the the error?  GPUd really detected the error, and we should report it, just need to 
1. make the error human-readable to let users know what the issue is.
2. currently, we can skip checking if the mountpoint is a subpath of a container.